internal/pkg/utils/testproject/project.go

```diff
@@ -342,6 +342,12 @@ func (p *Project) createConfigsRequests(configs []*model.ConfigWithRows, pool *c
 		if request, err := p.storageApi.CreateConfigRequest(config); err == nil {
 			branch := p.branchesById[config.BranchId]
 			p.logf("creating config \"%s/%s/%s\"", branch.Name, config.ComponentId, config.Name)
+			request.OnSuccess(func(response *client.Response) {
+				if len(config.Config.Metadata) > 0 {
+					request := p.StorageApi().UpdateConfigMetadataRequest(config.Config)
```

Comment:
:+1: to by sme takto mohli robit aj priamo s metadatami, bez pouzitia mappera :)
... ak bude cas, tak mozeme na konci sprintu na to pozriet.